docs/language/operators/over.md

```diff
@@ -22,8 +22,8 @@ generated from each such result depending on its types:
 entry in the map, and
 * all other values generate a single value equal to itself.
 
-Records can be converted to maps with the [`flatten` function](../functions/flatten.md)
-resulting in a map that can be traversed,
+A record can be converted with the [`flatten` function](../functions/flatten.md) to
+an array that can be traversed,
```

Comment:
The word "map" seemed inappropriate here, so I reworded. I think I see the spirit of what was being said before because the breaking out of the separately-named `key` and `value` portions allow for treating these in map-like ways, but since we already have an official `map` type (not to mention the `map` function), I figured it'd be good to avoid lumping on yet another use of the word "map".